java/src/main/java/com/powsybl/dataframe/network/extensions/SecondaryVoltageControlDataframeProvider.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2021-2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dataframe.network.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.dataframe.network.ExtensionInformation;
+import com.powsybl.dataframe.network.NetworkDataframeMapper;
+import com.powsybl.dataframe.network.NetworkDataframeMapperBuilder;
+import com.powsybl.dataframe.network.adders.NetworkElementAdder;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.extensions.SecondaryVoltageControl;
+import com.powsybl.iidm.network.extensions.SecondaryVoltageControl.ControlZone;
+
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * @author Hugo Kulesza <hugo.kulesza@rte-france.com>
+ */
+@AutoService(NetworkExtensionDataframeProvider.class)
+public class SecondaryVoltageControlDataframeProvider implements NetworkExtensionDataframeProvider {
+
+    @Override
+    public String getExtensionName() {
+        return SecondaryVoltageControl.NAME;
+    }
+
+    @Override
+    public ExtensionInformation getExtensionInformation() {
+        return new ExtensionInformation(SecondaryVoltageControl.NAME,
+                "Provides information about the secondary voltage control zones and units",
+                "index : ");
```

Comment:
I think it would be great to add that 2 dataframes are needed and their attributes definitions